src/alert/alert.ts

```diff
@@ -60,11 +60,16 @@ export class NgbAlert {
  * Alerts that can be dismissed without any additional code.
  */
 @Directive({selector: 'template[ngbAlert]'})
-export class NgbDismissibleAlert implements OnInit, OnDestroy {
+export class NgbSelfClosingAlert implements OnInit, OnDestroy {
   private _popupService: PopupService<NgbAlert>;
   private _timeout;
 
   /**
+   * A flag indicating if the alert can be dismissed (closed) by a user. If this flag is set, a close button (in a
+   * form of a cross) will be displayed.
```

Comment:
ok, i'm confused.  how are these lines showing up as "new" when they are already present in [the original file](https://github.com/ng-bootstrap/ng-bootstrap/blob/master/src/alert/alert.ts#L38-L41)?